.github/workflows/pr-commands.yaml

```diff
@@ -1,55 +1,75 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
 on:
   issue_comment:
     types: [created]
+
 name: Commands
+
 jobs:
   document:
-    if: startsWith(github.event.comment.body, '/document')
+    if: ${{ github.event.issue.pull_request && (github.event.comment.author_association == 'MEMBER' || github.event.comment.author_association == 'OWNER') && startsWith(github.event.comment.body, '/document') }}
     name: document
-    runs-on: macOS-latest
+    runs-on: ubuntu-latest
```

Comment:
out of interest, why the change of os?

Reply:
(Sorry -- I responded this before but apparently I did not hit "Comment".)

I don't know. I brainlessly copied the r-lib workflow via `usethis::use_github_action_pr_commands()`.